### ISSUE TITLE
dependency conflict when adding `apropos-tx` to `plutus-scaffold`-derived project.

### ISSUE THREAD

**Issue by peter-mlabs** [mlabs-haskell/apropos-tx#27]
Adding the lines 
```
  inputs.apropos-tx.url = "github:mlabs-haskell/apropos-tx";
(...)
extraSources = (...) ++ [ { src = inputs.apropos-tx; subdirs = [ "." ]; } ]
```
in the appropriate places of `plutus-scaffold` leads to the following dependency error:

```
cabal: Could not resolve dependencies:
[__0] trying: apropos-tx-1.0 (user goal)
[__1] next goal: base (dependency of apropos-tx)
[__1] rejecting: base-4.16.0.0/installed-4.16.0.0 (conflict: apropos-tx =>
base^>=4.14)
[__1] skipping: base-4.16.0.0, base-4.15.1.0, base-4.15.0.0 (has the same
characteristics that caused the previous version to fail: excluded by
constraint '^>=4.14' from 'apropos-tx')
[__1] rejecting: base-4.14.3.0, base-4.14.2.0, base-4.14.1.0, base-4.14.0.0,
base-4.13.0.0, base-4.12.0.0, base-4.11.1.0, base-4.11.0.0, base-4.10.1.0,
base-4.10.0.0, base-4.9.1.0, base-4.9.0.0, base-4.8.2.0, base-4.8.1.0,
base-4.8.0.0, base-4.7.0.2, base-4.7.0.1, base-4.7.0.0, base-4.6.0.1,
base-4.6.0.0, base-4.5.1.0, base-4.5.0.0, base-4.4.1.0, base-4.4.0.0,
base-4.3.1.0, base-4.3.0.0, base-4.2.0.2, base-4.2.0.1, base-4.2.0.0,
base-4.1.0.0, base-4.0.0.0, base-3.0.3.2, base-3.0.3.1 (constraint from
non-upgradeable package requires installed instance)
[__1] fail (backjumping, conflict set: apropos-tx, base)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, apropos-tx
```

**Comment by peter-mlabs** [comment]
Bumping the version to 4.16 causes a dependency to  fail for `Win32-network`.

**Comment by jhodgdev** [comment]
Will be closed by #37
